modules/lib/maintainers.nix

```diff
@@ -201,6 +201,12 @@
     github = "MForster";
     githubId = 4067975;
   };
+  michaelCTS = {
+    name = "michaelCTS";
+    email = "michael.vogel@cts.co";
+    github = "michaelCTS";
+    githubId = 132582212;
+  };
```

Comment:
No need to add yourself here if you already is in the Nixpkgs `maintainer-list.nix`.
```suggestion
```

Reply:
Removed and force-pushed